docs/pywbemcligeneraloptions.rst

```diff
@@ -580,49 +612,49 @@ PYWBEMCLI_MOCK_SERVER           ``--mock-server``
 PYWBEMCLI_LOG                   ``--log``
 ==============================  ============================
 
-If any of these environment variables are set, the corresponding general
-options on the command line are not required and the value of the environment
-variable is used. If both the environment variable and the command line option
-are included the command line option overrides the environment variable with no
+If these environment variables are set, the corresponding general options
+default to the value of the environment variables.
+
```

Comment:
We have several one sentence paragraphs here.  Lets group the first 3 sentences into a single paragraph.

Reply:
DONE